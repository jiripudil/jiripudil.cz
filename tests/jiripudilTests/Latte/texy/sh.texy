## Title of the document

Some paragraph.

/--sh
$ ./configure && make && make install
\--
