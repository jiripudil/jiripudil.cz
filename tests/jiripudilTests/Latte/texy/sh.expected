
<h2>Title of the document</h2>

<p>Some paragraph.</p>

<pre><code>$ ./configure &amp;&amp; make &amp;&amp; make install</code></pre>
